npm/private/npm_translate_lock.bzl

```diff
@@ -390,9 +390,6 @@ To disable this warning, set `warn_on_unqualified_tarball_url` to False in your
         ))
     return result
 
-def _join(*elements):
```

Comment:
again, I think this is broken because theirs doesn't discard empty segments like when workspace name is "" because it's the local one

Reply:
I think it does reading the impl, https://github.com/bazelbuild/bazel-skylib/blob/main/lib/paths.bzl#L88

The only way it could go terribly wrong is if you pass `None` values to it instead of empty strings.